eox_tagging/edxapp_accessors.py

```diff
@@ -4,14 +4,16 @@
 from django.contrib.auth.models import User
 from django.contrib.sites.models import Site
 from django.core.exceptions import ValidationError
-from eox_core.edxapp_wrapper.enrollments import get_enrollment
-from eox_core.edxapp_wrapper.users import get_edxapp_user
 from opaque_keys.edx.keys import CourseKey
 
-try:
-    from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
-except ImportError:
-    from eox_tagging.test.test_utils import CourseOverview  # pylint: disable=ungrouped-imports, useless-suppression
+from eox_tagging.edxapp_wrappers.backends.course_overview_i_v1 import get_course_overview
```

Comment:
you shouldn't call the backend's files, this must be something like this

```suggestion
from eox_tagging.edxapp_wrappers.course_overview import get_course_overview
```